Helm/opensearch-dashboards/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "opensearch-dashboards.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Should this `-}}` on the `if` and `else` statements be just `}}` to match the rest of the template?

Reply:
This does not fall on the conventions for the options in this template. In all the places here I have used `-}}`.